--- biewlib/sysdep/x86_64/cpu_info.c.orig	2019-02-24 21:31:50.000000000 +0200
+++ biewlib/sysdep/x86_64/cpu_info.c	2019-02-24 21:33:03.000000000 +0200
@@ -394,8 +394,8 @@
       "	fstp	%%st(1)\n"
       "	fbld	8(%1)\n"
       "	fbstp	8(%1)\n"
-      "	fild	(%1)\n"
-      "	fistp	(%1)\n"
+      "	fildl	(%1)\n"
+      "	fistpl	(%1)\n"
       "	fldt	4(%1)\n"
       "	fstpt	4(%1)\n"
       "	fstp	%%st(1)\n"
@@ -434,8 +434,8 @@
       "	fstp	%%st(1)\n"
       "	fyl2xp1\n"
       "	fstp	%%st(1)\n"
-      "	fild	(%1)\n"
-      "	fistp	(%1)"
+      "	fildl	(%1)\n"
+      "	fistpl	(%1)"
      ::"r"(counter),"r"(arr18bytes)
      :"st","st(1)","st(2)","st(3)");
      retval++;
